static/src/javascripts/projects/common/modules/identity/api.ts

```diff
@@ -168,7 +170,73 @@ export const buildNewsletterUpdatePayload = (
 	return newsletter;
 };
 
+type SignedOutWithCookies = { kind: 'SignedOutWithCookies' };
+export type SignedInWithCookies = { kind: 'SignedInWithCookies' };
+type SignedOutWithOkta = { kind: 'SignedOutWithOkta' };
+export type SignedInWithOkta = {
+	kind: 'SignedInWithOkta';
+	accessToken: AccessToken<never>;
+	idToken: IDToken<CustomIdTokenClaims>;
+};
+
+export type AuthStatus =
+	| SignedOutWithCookies
+	| SignedInWithCookies
+	| SignedOutWithOkta
+	| SignedInWithOkta;
+
+const isInOktaExperiment = () => {
```

Comment:
Is there a reason to have this as a method? Perhaps to check at a certain time rather than at runtime?

Reply:
no reason! I think a constant would be fine here, neither parts of the config should change once the page is loaded